src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -122,9 +122,9 @@ def show_version(self):
         print(ver_string)
         show_updates(updates_available_components)
 
-    def exception_handler(self, ex):  # pylint: disable=no-self-use
+    def exception_handler(self, ex, command):  # pylint: disable=no-self-use
```

Comment:
Please make the newly added parameter as optional, otherwise it will affect other callers
```suggestion
    def exception_handler(self, ex, command=None):  # pylint: disable=no-self-use
```

Reply:
Done.